src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/netstandard1.0/Microsoft.NET.Sdk.Functions.Publish.ZipDeploy.targets

```diff
@@ -0,0 +1,49 @@
+﻿<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Functions.Publish.ZipDeploy.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
```

Comment:
You can add another property IsZipDeploySupported to true & use it in the UI to determine if this feature should be enabled.